lib/wallet.js

```diff
@@ -315,6 +315,74 @@ class WalletClient extends Client {
     return this.get(`/wallet/${id}/coin/${hash}/${index}`);
   }
 
+  /**
+   * Get NameState from wallet
+   * @param {String} id
+   * @param {String} name
+   * @returns {Promise}
+   */
+
+  getName(id, name) {
+    return this.get(`/wallet/${id}/name`, { name });
+  }
+
+  /**
+   * Get all names wallet is managing
+   * @param {String} id
+   * @returns {Promise}
+   */
+
+  getNames(id) {
+    return this.get(`/wallet/${id}/names`);
+  }
+
+  /**
+   * Get NameState from wallet including
```

Comment:
Maybe we should start with "Get auction info for the provided name. This includes: name info, bids and reveals."